docs/core/integration-quality-scale/rules/iqs051.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "IQS051 - The documentation provides troubleshooting information"
+---
+
+IQS051 is as follows:
+
+The documentation provides troubleshooting information
+
+## Reasoning
+
+We should give the user enough information to figure out themselves on how to fix a common issue.
+This decreases the amount of support requests and increases the user experience.
```

Comment:
```suggestion

We should provide instructions on how to fix a common issue. If possible, a troubleshooting topic should include a description of the symptom and the steps needed to fix the situation.
This decreases the amount of support requests and increases the user experience.
```

Reply:
Ugh, it's a very dutch translation mistake I keep making